docs/user/lectures.rst

```diff
@@ -0,0 +1,164 @@
+.. _lectures:
+
+Lectures
+========
+
+.. contents:: Content of this document
+    :local:
+    :depth: 2
+
+Overview
+--------
+Apart from :ref:`Exercises` and :ref:`Exams`, Artemis can also host the **Lectures** of a course.
+
+Students
+^^^^^^^^
+
+Students can see all published lectures of a course, grouped by date, by clicking on the tab labeled *Lectures*.
+
+|lectures-overview|
+
+Clicking on a lecture leads to following page, where students can see the title, description, lecture units, and attachments.
+
+|lecture-details|
+
+Instructors
+^^^^^^^^^^^
+Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings, and exercises into lectures, and define learning goals.
```

Comment:
```suggestion
Instructors can upload lecture slides, divide lectures into units, integrate video streams, lecture recordings, and exercises into lectures and define learning goals.
```

Reply:
```suggestion
Instructors can upload lecture slides, integrate video streams, and lecture recordings.
This material can be grouped into logical lecture units.
To directly link the the necessary knowledge to its application, regular course exercises can be made part of lectures, too.
Instructors can also define learning goals so that students can easier keep track of the knowledge they should have after working with those lecture materials.
```

I think it would be worth elaborating more on that single sentence anyway as it combines a lot of different elements of lectures into one.
The suggestion is just a quick write-up of the four main elements that I suppose the sentence tries to convey. I’m sure they can be extended or reworded in a better way.